Client/Assets/scrape_plugins.py

```diff
@@ -26,7 +26,7 @@ def main():
             shutil.move(downloadedFile, os.path.join(directory, file))
 
 def getLocaleList():
-    response = requests.get('https://l10n.mozilla-community.org/~flod/webstatus/api/?product=firefox-ios&txt')
+    response = requests.get('https://hg.mozilla.org/releases/mozilla-aurora/raw-file/default/browser/locales/all-locales')
```

Comment:
This doesn't make sense: why are you switching from the list of locales that are translating Firefox for iOS to the list of locales with builds for **Firefox desktop** (and then scrap the searchplugins from the mobile folder used in **Firefox for Android**?